app/pages/shows/_slug/_id.vue

```diff
@@ -126,9 +126,38 @@ export default {
       airtimeAsc: true
     }
   },
-  fetch () {
-    this.fetchEpisodeData()
-  },
+  async fetch () {
+      //Fetch episode data
+      await this.$axios.get(`${arcsiBaseURL}/show/${this.slug}/item/${this.id}`)
```

Comment:
Sorry for the stupid question but this and the URL at line 138 and 152 are the ones which we use to reach the Arcsi API, am I right?
I suppose these parameters (this.slug and this.id) in these URLs are route params, but I don't understand properly how and where from we get them. Could you explain that to me?

Reply:
It's Nuxt logic and it comes from the path of the component, in this case `\app\pages\shows\_slug\_id.vue`, so basically these're template assets. Thanks for the review!

Reply:
I should dive a bit deeper into these Nuxt stuff!